packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.base.tsx

```diff
@@ -127,12 +127,13 @@ export class BreadcrumbBase extends BaseComponent<IBreadcrumbProps, any> {
     // Find index of last rendered item so the divider icon
     // knows not to render on that item
     const lastItemIndex = renderedItems.length - 1;
+    const hasOverflowItems = renderedOverflowItems && renderedOverflowItems.length !== 0;
 
     const itemElements: JSX.Element[] = renderedItems.map(
       (item, index) => (
         <li className={ this._classNames.listItem } key={ item.key || String(index) }>
           { onRenderItem(item, this._onRenderItem) }
-          { index !== lastItemIndex && (
+          { (index !== lastItemIndex || (hasOverflowItems && index === overflowIndex! - 1)) && (
```

Comment:
mind adding some parenthesis to ensure your booleans are evaluating what to expect?

```
&& index === (overflowIndex - 1)
```

Also you shouldn't need to add a ! after overflowIndex; should it default to a value? Or should you check for undefined?

Reply:
We have the overflowIndex as default props but the default props is not working with optional props.  Compiler complains Object is possibly 'undefined'